web/src/pages/api/v1/graphql.ts

```diff
@@ -25,6 +26,7 @@ export default async function handleGraphQL(
   }
 
   const authorization = req.headers["authorization"]?.replace("Bearer ", "");
+  const team_id = req.headers.team_id as string | undefined;
```

Comment:
we're not validating the `team_id` here which is fine, but we need to document thoroughly that validation must occur in Hasura at the permissions level, to prevent someone accessing a team to which they're not a member